conans/test/unittests/tools/files/collect_lib_test.py

```diff
@@ -43,21 +43,6 @@ def test_collect_libs():
     result = collect_libs(conanfile, folder="custom_folder")
     assert ["customlib"] == result
 
-    # Warn same lib different folders
-    conanfile = ConanFileMock()
-    conanfile.folders.set_base_package(temp_folder())
-    conanfile.cpp_info = CppInfo(conanfile.name, "")
-    custom_mylib_path = os.path.join(conanfile.package_folder, "custom_folder", "mylib.lib")
-    lib_mylib_path = os.path.join(conanfile.package_folder, "lib", "mylib.lib")
-    save(custom_mylib_path, "")
-    save(lib_mylib_path, "")
-    conanfile.cpp_info.libdirs = ["lib", "custom_folder"]
-    result = collect_libs(conanfile)
-    assert ["mylib"] == result
-    assert "Library 'mylib' was either already found in a previous "\
```

Comment:
Why this test is being removed? Maybe it is fine to remove the warning check, but this test should still work and ``result == ["mylib"]``?
Also, it is also that necessary to remove the warning? If ``collect_libs()`` is decided to use one specific name in a group of related ones, isn't it good to show some information about this?

Reply:
Yes, the first assert is still valid. The second assert fails if I keep the warning, because now there is a first pass grouping the same libs in all folders, either symlinks or real file, and it loses the capability to track duplicated real files. Displaying a warning for symlinks is unnecessary noise. I can try to track duplicated real files, but I'm not sure it's worth the effort and overhead.

Reply:
I'll add tests for symlinks

Reply:
Good, lets keep the first asset, and I agree, not worth the effort to keep the warning, it can be removed.